src/UI/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -10,6 +10,7 @@
     <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
     <RazorCompileOnBuild>true</RazorCompileOnBuild>
     <ResolvedRazorCompileToolset>RazorSdk</ResolvedRazorCompileToolset>
```

Comment:
Is this still required? We might already have a bug tracking that.

Reply:
It's hidden behind the flag that needs to be cleaned up. Need one more round of stuff to get in to the CLI to sort those out.